webapp/appModules/home/components/surveysList.js

```diff
@@ -21,18 +19,18 @@ const SurveyListHeader = () => (
   </div>
 )
 
-const SurveyRow = ({survey, surveyInfo, setActiveSurvey}) => {
-  const surveyId = Survey.getSurveyId(survey)
+const SurveyRow = ({surveyInfoRow, surveyInfo, setActiveSurvey}) => {
```

Comment:
why is the parameter called "surveyInfoRow" and not just "surveyInfo"?

Reply:
because surveyInfo is the state object.